README.md

```diff
@@ -81,6 +81,8 @@ gh gei generate-script --github-source-org "source-ghes-org" --github-target-org
 
 This will generate a new file (named `./migrate.ps1` by default) which gives you an opportunity to review the steps GEI-CLI will take. Once ready, running the `./migrate.ps1` file will kick off the migration!
 
+After running the migration and validating the migration data, we recommend logging in to your Azure storage account and deleting the blob container that was created and used to perform the migration.
```

Comment:
```suggestion
After running the migration and validating the migration data, you will need to remove the archive data from your Azure storage account. The data can either be deleted from the blob container or, if you're done all of your migrations, the blob container itself can be deleted. 
```

Reply:
👍  FYI we create a blob container per single migration

Reply:
Updated in https://github.com/github/gh-gei/pull/251/commits/63e18f058f37ca6aa529ed2af462be6c38410104